src/index.css

```diff
@@ -1,3 +1,53 @@
-body {
-  margin: auto;
+#root {
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  margin-top: 10px;
+}
+
+.calculator {
+  width: 700px;
+  display: flex;
+  flex-direction: column;
+}
+
+.display {
```

Comment:
This should use flex display 
![see](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/useFlex.png)